app/v1/controllers/LiabilitiesController.scala

```diff
@@ -57,6 +60,9 @@ class LiabilitiesController @Inject()(val authService: EnrolmentsAuthService,
         logger.info(message = s"${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] " +
           s"Successfully retrieved Liabilities from DES")
 
+        auditService.auditEvent(AuditEvents.auditReturns(serviceResponse.correlationId,
```

Comment:
should be "auditLiability" not "auditReturns" right?

Reply:
sorry